keyboards/the_royal/sukari/keymaps/default/keymap.c

```diff
@@ -0,0 +1,173 @@
+),
+
+
+[_RAISE] = LAYOUT_true(
+    KC_F1,     KC_F2,     KC_F3,     KC_F4,     KC_F5,     KC_F6,     KC_F7,     KC_F8,     KC_F9,     KC_F10,    KC_F11,    KC_F12,
+    _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,
+    KC_CAPS,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_LCBR,   KC_RCBR,   KC_PIPE,   KC_PLUS,
+    _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_LBRC,   KC_RBRC,   KC_BSLS,   KC_EQUAL,
+    _______,         _______,                                    _______,                                    _______,        _______
+),
+
+
+[_OTHER] = LAYOUT_true(
+    TD(TD_RST),   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,    _______,
+    _______,      _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,    _______,
+    _______,      _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_UP,     _______,    _______,
+    _______,      _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_LEFT,   KC_DOWN,   KC_RIGHT,   _______,
+    KC_PSCR,            KC_LGUI,                                    _______,                                    _______,         _______
+),
+
+
+[_GAME] = LAYOUT_true(
+    KC_ESC,    KC_1,   KC_2,   KC_3,   KC_4,   KC_5,   KC_6,   KC_7,   KC_8,      KC_9,     KC_0,      KC_MINUS,
+    KC_TAB,    KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,      KC_O,     KC_P,      KC_BSPC,
+    KC_LCTL,   KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,      KC_L,     KC_SCLN,   KC_QUOT,
+    KC_LSFT,   KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,   KC_DOT,   KC_SLSH,   KC_ENTER,
+    KC_DEL,       KC_LALT,                       KC_SPACE,                             KC_LEFT,        KC_RIGHT
+)
+
+};
+
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+        case LOWER:
+          if (record->event.pressed) {
+            layer_on(_LOWER);
+            update_tri_layer(_LOWER, _RAISE, _OTHER);
+          } else {
+            layer_off(_LOWER);
+            update_tri_layer(_LOWER, _RAISE, _OTHER);
+          }
+          return false;
+            break;
+        case RAISE:
+          if (record->event.pressed) {
+            layer_on(_RAISE);
+            update_tri_layer(_LOWER, _RAISE, _OTHER);
+          } else {
+            layer_off(_RAISE);
+            update_tri_layer(_LOWER, _RAISE, _OTHER);
+          }
+          return false;
+            break;
+    }
+  return true;
+};
+
+
+#ifdef RGBLIGHT_ENABLE
+// START - RGB Indicator LED Declarations ----------
+layer_state_t layer_state_set_user(layer_state_t state) {
```

Comment:
You should probably call this from `keyboard_post_init_user` to set this by default. 

Eg:
```suggestion
void keyboard_post_init_user(void) {
    layer_state_set_user(0);
}

layer_state_t layer_state_set_user(layer_state_t state) {
```